js/binance.js

```diff
@@ -589,6 +590,127 @@ module.exports = class binance extends Exchange {
         return this.options['timeDifference'];
     }
 
+    async fetchCurrencies (params = {}) {
+        const response = await this.sapiGetCapitalConfigGetall (params);
```

Comment:
The problem with this endpoint is that it requires authentication, breaking public-api instances and preventing them from loading. It has to check if the user has specified the apiKey. Will add that shortly.